portfolio/src/main/webapp/index.html

```diff
@@ -8,21 +8,28 @@
   </head>
   <body>
     <div id="pageWrapper">
+      <div id="navbar">
+        <div class="navButton" onClick="highlightSection('basicInfo')">Introduction</div> 
+        <div class="navButton" onClick="highlightSection('hobbyWrapper')">Hobbies</div> 
+        <div class="navButton" onClick="highlightSection('projects')">Projects</div> 
+      </div> 
+
       <div id="introduction">
         <h1>Hi there!</h1>
         <div id="headshotWrapper">
-        <img id="headshot" src="city.jpg" alt="city skyline"/> 
+          <img class="headshot" id="image" src="city.jpg" alt="skyline of Beijing"/>
```

Comment:
I would recommend that you change the id from image to something like headshotImage. IDs need to be unique per page.

That said, as a reminder, you don't have to have ID for every HTML item.

Reply:
I deleted the ID for this image because I don't see any additional functionalities that I will add to this image by calling its unique ID.